modules/gitops-customizing-the-route.name.in.keycloak.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-customizing-the-route-name-in-keycloak_{context}"]
+= Customizing the route name in Keycloak
+
+Customize the route name of the Keycloak instance by adding a host name attribute in the Argo CD CR. Use this feature to support advanced routing use cases such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller shards].
+
+[id="prerequisites_customizing-the-route-name-in-keycloak"]
+.Prerequisites
+* You have installed Red Hat SSO on the cluster.
+* You have installed the {gitops-title} Operator in your cluster.
+* You have installed Argo CD on the cluster.
+
+.Procedure
+
+Perform the following steps to customize route in Keycloak.
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Click the *Argo CD* tab and select *{gitops-title}* from the installed Operators list.
+
+. Click the *YAML* tab and enter the following snippet to add the *host* key. 
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  namespace: sso.test
+[...]
+spec:
+  server:
+    host: sso.test.example.com
+    route:
+      enabled: true 
+      sso:
+    provider: keycloak
+      host: sso.test.example.com
+----
+
+. Click *Save*.
+
+. Click *Home* -> *Search* and enter `Route`.
+
+. Select the *keycloak* route. 
+
+. Click the *YAML* tab.
++
+You can view the following `host` key is added in the *keycloak* route. 
++
+[source,yaml]
+----
+spec:
+  sso:
+    provider: keycloak
+    keycloak:
+     verifyTLS: false
+     host: sso.test.example.com
+----
```

Comment:
there is no sso key in  Route CR spec field, you can change this to :
```
kind: Route
metadata:
  name: keycloak
  labels:
    application: keycloak
spec:
  host: sso.test.example.com
status:
  ingress:
    - host: sso.test.example.com
```

Reply:
Fixed.